bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -313,6 +315,36 @@
    @XmlElement
    private String                timeZoneId;
 
+   /**
+    * An array of {@link TourTimerPause}
+    * A timer pause is a device event triggered by the user.
+    * TODO FB rename to ?
+    * tourDevicePauses
+    * tourUserPauses
+    * .....
+    */
+   //TODO FB Can't i map it just with the tourid ???
+   @OneToMany(fetch = FetchType.EAGER, cascade = ALL, mappedBy = "tourData")
+   @Cascade(org.hibernate.annotations.CascadeType.DELETE_ORPHAN)
+   @XmlElementWrapper(name = "TourTimerPauses")
+   @XmlElement(name = "TourTimerPause")
+   private List<TourTimerPause>             tourTimerPauses                         = new ArrayList<>();
+
+   /**
+    * Total paused time in seconds
+    *
+    * This number could come from a direct value or from {@link tourTimerPauses}
+    */
+   @XmlElement
+   private long                  tourPausedTime;
+
+
+   /**
+    * Total recorded time in seconds
+    */
+   @XmlElement
+   private long                  tourRecordedTime;
```

Comment:
There is `tourRecordedTime` and `tourRecordingTime`, you already commented to rename it. But when you do a renaming action, for me it would be helpful that these 5 time fields have a postfix to identify them easily if the time is from a `Device` or it is `Computed`, e.g.

```
tourElapsedTime_Device
tourRecordedTime_Device
tourPausedTime_Device

tourMovingTime_Computed
tourBreakTime_Computed
```

Reply:
Maybe rename them to

```
tourDeviceTime_Elapsed
tourDeviceTime_Recorded
tourDeviceTime_Paused

tourComputedTime_Moved
tourComputedTime_Break
```

Reply:
I like those names except this one :
`tourComputedTime_Moved`

I would rename to 
`tourComputedTime_Mov**ing**`

Reply:
This is a no-go to not use proper english, I'm wondering why I suggested `Moved` because in the code, `Moving`Time is already everywere.
During the code review, I also used the `ed` "postfix" of the new time fields to identify device time values.

Reply:
Ok, so do we agree that this is the correct and final property names ?

```
tourDeviceTime_Elapsed
tourDeviceTime_Recorded
tourDeviceTime_Paused

tourComputedTime_Moving
tourComputedTime_Break
```

If it's ok with you, I will rename them for the last time, including renaming the fields in the DB

Reply:
YES, the names are OK

Reply:
Fixed